plotnine/geoms/annotation_alternating_background.py

```diff
@@ -0,0 +1,127 @@
+from matplotlib.collections import PolyCollection
+import pandas as pd
+import numpy as np
+
+from ..utils import to_rgba, SIZE_FACTOR
+from ..doctools import document
+from ..layer import layer
+from ..coords import coord_flip
+from ..scales.scale import scale_discrete
+from .geom import geom
+from .annotate import annotate
+
+
+@document
+class annotation_alternating_background(annotate):
+    """
+    Alternating backgrounds, centered around each label on x-axis.
+    (Use coord_flip for y-axis)
+
+    Useful for geom_jitter.
+
+    {usage}
+    plot += annotation_alternating_background(fill=['red', 'blue'], alpha=0.3)
+
+    Parameters
+    ----------
+    fills - list of colors to alternate through.
+    Default: ["#AAAAAA", "#CCCCCC"]
+
+    {common_parameters}
+    """
+
+    def __init__(self, **kwargs):
+        self._annotation_geom = _geom_annotation_alternating_background(
+            **kwargs)
+
+    def __radd__(self, gg, inplace=False):
+        # background is always the lowest layer
+        l = layer.from_geom(self._annotation_geom)
+        l._insert(0, gg)
+        return gg
+
+
+class _geom_annotation_alternating_background(geom):
+
+    DEFAULT_AES = {}
+    REQUIRED_AES = set()
+    DEFAULT_PARAMS = {
+        "stat": "identity",
+        "position": "identity",
+        "na_rm": False,
+        "color": None,
+        "fills": ["#AAAAAA", "#CCCCCC"],
+        "linetype": "solid",
+        "size": 0,
+        "alpha": 0.5,
+    }
+    legend_geom = "polygon"
+
+    @staticmethod
+    def draw_group(data, panel_params, coord, ax, **params):
```

Comment:
On the implementation, this `geom` should create the right data and leave the rest to `geom_rect`.